cnd/src/quickcheck.rs

```diff
@@ -238,17 +237,19 @@ impl Arbitrary for Quickcheck<ledger::Ethereum> {
     }
 }
 
-impl<AL, BL, AA, BA> Arbitrary for Quickcheck<Request<AL, BL, AA, BA>>
+impl<AL, BL, AA, BA, AI, BI> Arbitrary for Quickcheck<Request<AL, BL, AA, BA, AI, BI>>
 where
     AL: Ledger,
```

Comment:
I think these bounds can go away now. Please also check in other places, where we previously only set them to access `L::Identity`.